core/src/test/java/apoc/ApocSplitTest.java

```diff
@@ -0,0 +1,473 @@
+        "apoc.log.stream",
+        "apoc.text.phoneticDelta",
+        "apoc.export.arrow.stream.all",
+        "apoc.export.arrow.stream.graph",
+        "apoc.export.arrow.stream.query",
+        "apoc.export.arrow.all",
+        "apoc.export.arrow.graph",
+        "apoc.export.arrow.query",
+        "apoc.export.cypher.all",
+        "apoc.export.cypher.data",
+        "apoc.export.cypher.graph",
+        "apoc.export.cypher.query",
+        "apoc.export.cypher.schema",
+        "apoc.import.json",
+        "apoc.export.json.all",
+        "apoc.export.json.data",
+        "apoc.export.json.graph",
+        "apoc.export.json.query",
+        "apoc.import.csv",
+        "apoc.export.csv.all",
+        "apoc.export.csv.data",
+        "apoc.export.csv.graph",
+        "apoc.export.csv.query",
+        "apoc.import.graphml",
+        "apoc.export.graphml.all",
+        "apoc.export.graphml.data",
+        "apoc.export.graphml.graph",
+        "apoc.export.graphml.query",
+        "apoc.spatial.sortByDistance",
+        "apoc.spatial.geocodeOnce",
+        "apoc.spatial.geocode",
+        "apoc.spatial.reverseGeocode",
+        "apoc.create.node",
+        "apoc.create.addLabels",
+        "apoc.create.setProperty",
+        "apoc.create.setRelProperty",
+        "apoc.create.setProperties",
+        "apoc.create.removeProperties",
+        "apoc.create.setRelProperties",
+        "apoc.create.removeRelProperties",
+        "apoc.create.setLabels",
+        "apoc.create.removeLabels",
+        "apoc.create.nodes",
+        "apoc.create.relationship",
+        "apoc.create.vNode",
+        "apoc.create.vNodes",
+        "apoc.create.vRelationship",
+        "apoc.create.virtualPath",
+        "apoc.create.clonePathToVirtual",
+        "apoc.create.clonePathsToVirtual",
+        "apoc.create.uuids",
+        "apoc.warmup.run",
+        "apoc.stats.degrees",
+        "apoc.help",
+        "apoc.refactor.rename.label",
+        "apoc.refactor.rename.type",
+        "apoc.refactor.rename.nodeProperty",
+        "apoc.refactor.rename.typeProperty",
+        "apoc.refactor.extractNode",
+        "apoc.refactor.collapseNode",
+        "apoc.refactor.cloneNodes",
+        "apoc.refactor.cloneSubgraphFromPaths",
+        "apoc.refactor.cloneSubgraph",
+        "apoc.refactor.mergeNodes",
+        "apoc.refactor.mergeRelationships",
+        "apoc.refactor.setType",
+        "apoc.refactor.to",
+        "apoc.refactor.invert",
+        "apoc.refactor.from",
+        "apoc.refactor.normalizeAsBoolean",
+        "apoc.refactor.categorize",
+        "apoc.refactor.deleteAndReconnect",
+        "apoc.convert.setJsonProperty",
+        "apoc.convert.toTree",
+        "apoc.neighbors.tohop",
+        "apoc.neighbors.tohop.count",
+        "apoc.neighbors.byhop",
+        "apoc.neighbors.byhop.count",
+        "apoc.neighbors.athop",
+        "apoc.neighbors.athop.count");
+
+    public static final Set<String> CORE_FUNCTIONS = Set.of("apoc.temporal.format",
```

Comment:
Small detail but maybe "apoc.temporal.format" could be on new line as it is above